### ISSUE TITLE
tacks-style shebang support for dev/test without pipeline

### ISSUE THREAD

**Issue by yawn** [superwerker/superwerker#17]
Like this for `bin/runwith`: 

```
#!/bin/bash -Eeuo pipefail

TEMPLATE=$(mktemp)
trap "{ rm -f $TEMPLATE; }" EXIT
tail -n +2 "${!#}" > $TEMPLATE

eval "${@:1:$#-1}" "$TEMPLATE"
```

Then this works:

```
#! bin/runwith aws cloudformation deploy --stack-name test --template-file 
AWSTemplateFormatVersion: 2010-09-09
Transform: AWS::Serverless-2016-10-31
Description: Test

Resources:

  Test:
    Type: AWS::S3::Bucket
    Properties:
      BucketName: f5573031-92f9-437a-b52b-46f83c0d0f67
      PublicAccessBlockConfiguration:
        BlockPublicAcls: true
        BlockPublicPolicy: true
        IgnorePublicAcls: true
        RestrictPublicBuckets: true
```

All files must be `+x`.

**Comment by yawn** [comment]
A bit hacky but works just fine (TM) @s0enke

**Comment by yawn** [comment]
Closed via 7c94183f10e833973fde16fe8994dafc1e5b8a78
